.github/workflows/test_and_release.yml

```diff
@@ -0,0 +1,126 @@
+name: Test and Release
+
+on:
+  workflow_dispatch:
+  release:
+    types: [published]
+  pull_request:
+    branches:
+      - master
+      - v[0-9]+.[0-9]+-dev
+
+jobs:
+  test:
+    name: Run SDK tests
+    runs-on: ubuntu-20.04
+    timeout-minutes: 60
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
+        with:
+          node-version: '12'
+
+      - name: Enable NPM cache
+        uses: actions/cache@v2
+        with:
+          path: '~/.npm'
+          key: ${{ runner.os }}-node-${{ hashFiles('**/package-lock.json') }}
+          restore-keys: |
+            ${{ runner.os }}-node-
+
+      - name: Install NPM dependencies
+        run: npm ci
+
+      - name: Check NPM package lock version is updated
+        run: npm run check-package # TODO: does this exist as an action already?
+
+      - name: Cache Docker layers
+        uses: satackey/action-docker-layer-caching@v0.0.11
+        continue-on-error: true
+        with:
+          concurrency: 8
+
+      - name: Parse variables from PR body
+        uses: dashevo/gh-action-runtime-vars@master
+        id: runtime-vars
+        if: ${{ contains(github.event.pull_request.body, '/set-var')}}
+
+      - name: Set compatible branches for Platform components
+        uses: dashevo/gh-action-platform-branches@master
+        id: platform-branches
+        with:
+          override-major-version: 0
+          override-dashmate-branch: ${{ steps.runtime-vars.outputs.dashmate-branch }}
+          override-testsuite-branch: ${{ steps.runtime-vars.outputs.testsuite-branch }}
+
+      - name: Start Dash local network
+        uses: dashevo/gh-action-start-local-network@master
+        with:
+          actions-cache-url: ${{ steps.platform-branches.outputs.cache-url }}
+          actions-cache-token: ${{ steps.platform-branches.outputs.cache-token }}
+          drive-branch: ${{ steps.platform-branches.outputs.current-branch }}
+          dapi-branch: ${{ steps.runtime-vars.outputs.dapi-branch }}
+          dashmate-branch: ${{ steps.platform-branches.outputs.dashmate-branch }}
+          install-sdk-version: ${{ steps.runtime-vars.outputs.install-sdk-version }}
```

Comment:
It suppose to be the an install path for `npm i` command as I remember. Something like `github:dashevo/js-dash-sdk#${{ current branch }}`.

Reply:
I'm still not sure what this option does or why it also works without the SDK. But it looks like the logic you are requesting here already exists in `gh-action-start-local-network`: https://github.com/dashevo/gh-action-start-local-network/blob/master/bin/start-local-node.sh#L56-L60

Reply:
It has SDK but we kinda overriding it with a specific version. This logic exist not just for dashmate, but for test suite as well, but slightly different https://github.com/dashevo/platform-test-suite/blob/v0.19-dev/bin/test.sh#L87-L90. So you need to pass current branch as `sdk-branch` to `start-local-network` and `github:dashevo/js-dash-sdk#${{ current branch }}` to  `run-platfrom-test-suite`.

Reply:
I changed the way `gh-action-run-platform-test-suite` processes the variable [here instead](https://github.com/dashevo/gh-action-run-platform-test-suite/pull/1), so we don't need to make this change in all other repos using this action.

Reply:
Should we call then this input as `sdk-branch`?